src/lib.rs

```diff
@@ -1601,6 +1608,15 @@ mod test {
         use super::*;
         use num_traits::{Float, Pow};
 
+        #[test]
+        fn test_cis() {
+            assert!(close(Complex::cis(0.00 * f64::consts::TAU), _1_0i));
```

Comment:
Ah, `TAU` was only added to the standard library in 1.47. You could use [`FloatConst::TAU()`](https://docs.rs/num-traits/latest/num_traits/float/trait.FloatConst.html#method.TAU) in the meantime, or just switch back to `PI`.